flow/project.py

```diff
@@ -3184,38 +3623,39 @@ def classify(self, job):
         for _label in self.labels(job):
             yield _label
 
-    def completed_operations(self, job):
-        """Determine which operations have been completed for job.
+    def completed_operations(self, jobs):
+        """Determine which operations have been completed for
+        specified jobs.
 
         :param job:
-            The signac job handle.
+            Sequence of signac job handles.
         :type job:
-            :class:`~signac.contrib.job.Job`
+            Iterable
         :return:
             The name of the operations that are complete.
         :rtype:
             str
         """
         for name, op in self._operations.items():
-            if op.complete(job):
+            if op.complete(jobs):
                 yield name
 
-    def _job_operations(self, job, ignore_conditions=IgnoreConditions.NONE):
+    def _job_operations(self, jobs, ignore_conditions=IgnoreConditions.NONE):
         "Yield instances of JobOperation constructed for specific jobs."
         for name in self.operations:
             group = self._groups[name]
-            yield from group._create_run_job_operations(entrypoint=self._entrypoint, job=job,
+            yield from group._create_run_job_operations(entrypoint=self._entrypoint, jobs=jobs,
                                                         default_directives=dict(),
                                                         ignore_conditions=ignore_conditions,
                                                         index=0)
 
-    def next_operations(self, *jobs, ignore_conditions=IgnoreConditions.NONE):
+    def next_operations(self, jobs, ignore_conditions=IgnoreConditions.NONE):
```

Comment:
It seems like this function is identical to `_job_operations` now. Do we need `_job_operations` any more?

Reply:
No we don't. I'll remove `_job_operations`